content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -31,11 +31,11 @@ If the node where a Pod is running has enough of a resource available, it's poss
 allowed) for a container to use more resource than its `request` for that resource specifies.
 However, a container is not allowed to use more than its resource `limit`.
 
-For example, if you set a `memory` request of 256 MiB for a container, and that container is in
+For example, if you set a memory `request` of 256 MiB for a container, and that container is in
```

Comment:
The original is better. `memory` is literal value you would use in the API, but you would not put that in a field named `request`.

As you'll see from https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#resources the field is called `resources`, _plural_.

I do think the original text is better.